image/create.go

```diff
@@ -38,7 +38,7 @@ import (
 type ImageCreator struct {
 	Body         []byte
 	Version      ImageVersion
-	SigKeys      []sec.SignKey
+	SigKeys      []sec.PrivSignKey
```

Comment:
There is no such thing as `PubSignKey`, a sign key is private by definition.

Reply:
By "Pub" I mean the public component of the key; the part that you use to verify a signature.  Do you think `PrivSignKey` / `PubSignKey` is not a good way to name these types?

Reply:
`PubKey` would be ok for the verification, or `VerifyKey` (which sounds a bit weird!). I just mentioned it because saying that it is private is like saying: "look this number is prime, AND it also is only divisible by 1 and itself". All those name changes don't bring extra information. Although I would not mind if you prefer this way, since it's done already! ;-)

Reply:
I forgot to reply here in case others were following.  @utzig and I discussed this offline, and we concluded that while the current naming may not be the most intuitive, we couldn't think of any obviously-better candidates.  So, for now we will keep the current names.  If they prove too confusing, we can always change them later!